<launch>

  <!-- Number of fixes used to mean the coordinates of the base station -->
  <arg name="n_fix" default="6000"/>

  <!-- geodetic_utils node: set reference position -->
  <node pkg="geodetic_utils" type="set_gps_reference_node" name="set_gps_reference_node" args="$(arg n_fix) average" output="screen">
    <remap from="gps" to="/piksi/navsatfix_spp" />
  </node>

  <!-- Piksi node -->
  <node pkg="piksi_rtk_gps" type="piksi.py" name="piksi">
    <rosparam file="$(find piksi_rtk_gps)/cfg/piksi_driver_settings.yaml"/>
  </node>


</launch>
